build.gradle

```diff
@@ -46,7 +46,7 @@ dependencies {
 	compile("com.googlecode.json-simple:json-simple:1.1.1") {
 	   exclude group: 'junit'
 	}
-	compile group: 'com.vmware.singleton', name: 'singleton-i18n-patterns-core', version: '+'	
+	compile group: 'com.vmware.singleton', name: 'singleton-i18n-patterns-core', version: '0.+'
```

Comment:
Does 0.+ mean any version of 0.x?
If so, how about 0+? Does this mean any version bigger than 0?

Reply:
Yes.

According to nannan, change like this still can't resolve the problem, looks like we can't have '+' anywhere in pom file.
So I change the version to a fixed one.